### PR TITLE
ADAPT-7930 - focus on last element in focus stack on drawer hide

### PR DESCRIPTION
**Pull request by eleanor-heath**
Linked issue: https://github.com/adaptlearning/adapt_framework/issues/3042

### CODE REVIEW

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]
👀

File: src/core/js/views/drawerView.js [hunk 1]
```diff
@@ -211,7 +211,12 @@ class DrawerView extends Backbone.View {
     const direction = {};
     // only trigger popup:closed if drawer is visible
     if (this._isVisible) {
-      Adapt.a11y.popupClosed($toElement);
+      const focusStack = $(Adapt.a11y._popup._focusStack[0]);
+      const $elementToFocus = $toElement || focusStack;
+      if (!$elementToFocus) {
```
Comment (tomgreenfield) [inline src/core/js/views/drawerView.js:216]: This condition will never resolve to a truthy.

**Review by tomgreenfield — COMMENTED** [review]
I'm unable to replicate the original issue; this line should handle the focus element: https://github.com/adaptlearning/adapt_framework/blob/452e3d2dfb9aec4eac5cee2f120746ba17887aae/src/core/js/a11y/popup.js#L139

Please provide version information for Adapt and browser(s).

File: src/core/js/views/drawerView.js (hunk 1 above)
Comment (oliverfoster) [reply src/core/js/views/drawerView.js:216]: https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/a11y/popup.js#L139 This is what already happens in `Adapt.a11y.popupClosed` anyway

**Comment by eleanor-heath** [comment]
This issue on IE11 seems to be caused by the media component. To replicate add a few components e.g. text and graphic component, then add a media component below with a youtube link. Navigate through the items on the page using NVDA screen reader and press enter when the screenreader gets to the page level progress button. Navigate through all items until you get to the close button and click enter. You will not be able to navigate the page using the up and down arrows and the screen reader will read out "blank".

**Comment by oliverfoster** [comment]
@eleanor-heath did this get fixed elsewhere?

**Comment by eleanor-heath** [comment]
@oliverfoster no, Ive parked it for now until we test it against the latest framework before I carry on with the work.

**Pull request closed without merging**